### DISCUSSION TITLE
Audio input device not available

### DISCUSSION THREAD

**Discussion by manish-makwana** [monocasual/giada#606] [Q&A]
Hello, I've just downloaded giada for Windows (Win10 21H2). When I go to the settings to configure my audio device (shure condenser mic via an X2U usb interface) it's not available, with the input device option greyed out:

![image](https://user-images.githubusercontent.com/13429280/187061567-ab0d067a-fe32-466b-9f53-b0a1887177dc.png)

Ticking the box on the far right of the input device box doesn't change anything.

Any suggestions? Thanks!

FYI, I'd like to see if I can use giada for live music looping.

**Comment by gvnnz** [comment]
Hi @manish-makwana , make sure you tick the checkbox next to the input device drop-down menu. And then restart.

**Reply by jgusta** [reply]
Same issue on MacOS. Checking the box and restarting does nothing; the program reopens with the box unchecked.

<img width="460" alt="CleanShot 2022-09-03 at 13 59 24@2x" src="https://user-images.githubusercontent.com/2585228/188287506-414ea405-ac3a-4360-821f-6901e4801095.png">

**Comment by gvnnz** [comment]
@jgusta can yoy please paste here the log you get on startup? [Here](https://github.com/monocasual/giada/discussions/584) is how to enable console logging. Thanks!

**Reply by stebrun** [reply]
I have the same problem as described above. Is there any solution yet?
Here is what I get from the console log when starting Giada:

[MidiMapper::init] total midimaps found: 0
[Engine::init] MIDI map read failed!
[KA] using API 8
[KA] Opening device out=0, in=-1, samplerate=44100
[KA] 4 device(s) found
  0) Primõrer Soundtreiber
      ins=0 outs=2 duplex=0
      isDefaultOut=1 isDefaultIn=0
      sampleRates:
        4000 5512 8000 9600 11025 16000 22050 32000 44100 48000 88200 96000 176400 192000
  1) Lautsprecher (Realtek(R) Audio)
      ins=0 outs=2 duplex=0
      isDefaultOut=0 isDefaultIn=0
      sampleRates:
        4000 5512 8000 9600 11025 16000 22050 32000 44100 48000 88200 96000 176400 192000
  2) Primõrer Soundaufnahmetreiber
      ins=2 outs=0 duplex=0
      isDefaultOut=0 isDefaultIn=0
      sampleRates:
        11025 22050 44100 96000
  3) Microphone Array (Intel« Smart Sound Technologie f³r digitale Mikrofone)
      ins=2 outs=0 duplex=0
      isDefaultOut=0 isDefaultIn=0
      sampleRates:
        11025 22050 44100 96000
[mixer::reset] buffers ready - maxFramesInLoop=529200, framesInBuffer=1024
[mixer::enable] enabled
[KA] Start stream - latency = 0
[LangMapper::init] total langmaps found: 0
[init] Giada 0.24.0
[init] Build date: Mar 14 2023
[init] Release build
[init] Dependencies:
[init]   FLTK - 1.4.0
[init]   RtAudio - 6.0.0beta1
[init]   RtMidi - 5.0.0
[init]   Libsamplerate
[init]   Libsndfile - libsndfile-1.0.31
[init]   JSON for modern C++ - 3.10.5
[init]   JUCE - 7.0.1
[KA] Compiled RtAudio APIs: 3
  ASIO
  WASAPI
  DirectSound
[KM] Compiled RtMidi APIs: 1
  Microsoft Multimedia MIDI API

**Comment by gvnnz** [comment]
Please folks try again with the latest version 0.25 and see if the issue is still there. We reworked the whole Configuration panel, so things might have improved. Thanks!
